src/TodoListItem.jsx

```diff
@@ -0,0 +1,11 @@
+import item from "./TodoList";
+
+
+function TodoListItem (props) {
+    const { todo} = props;
```

Comment:
You can destructure the todo object in the same line of the function definition like this:
```suggestion
function TodoListItem ({todo}) {
```